mlir/test/Target/LLVMIR/Import/metadata-dereferenceable.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
+
+define void @deref(i64 %0, ptr %1) {
+  ; CHECK: dereferenceable = #llvm.dereferenceable<bytes = 4>
```

Comment:
I think you should also match the op itself and then do `CHECK-SAME`, just to sanity check that you are finding these on the ops that you expect to find them on